.github/workflows/build_lint.yml

```
@@ -32,6 +32,8 @@ jobs:
         os: [windows, ubuntu, macos]
         python-version: ["3.10"]
         include:
+          - os: ubuntu
+            python-version: "3.11"
```

Thanks, that needed to be done indeed! I would say let's bump the version that runs on all OSes to 3.11, and add a 3.10 job for Ubuntu.
```suggestion
        python-version: ["3.11"]
        include:
          - os: ubuntu
            python-version: "3.10"
```
Do you have any idea what the error is about? It doesn't look that Python 3.11 specific.

---

>  I would say let's bump the version that runs on all OSes to 3.11, and add a 3.10 job for Ubuntu.

Will do, though I'm still investigating. I have reproduced the bug in a Docker instance.

---

I found the source of the bug, but I'm not sure why the other Python didn't identify it as a bug actually

---

If you want to add the mod to this PR, I will close mine

---

Let's solve it in #1521, and limit the scope of this PR to updating the CI.

---

> Thanks, that needed to be done indeed! I would say let's bump the version that runs on all OSes to 3.11, and add a 3.10 job for Ubuntu.

@rht What do you think of this?

---

Sounds good. Doing it now.